ws_fnc/AI/fn_createGarrison.sqf

```diff
@@ -26,6 +27,8 @@ PARAMETERS:
 6. Array of classes to spawn                                                           | OPTIONAL - array w. strings  - default are classes defined below
 6. ALTERNATIVE: Faction to spawn                                                       | OPTIONAL - string - faction name
 7. Only garrison empty buildings                                                       | OPTIONAL - bool - true, if only empty buildings should be garrisoned. This is useful for overlappnig garrison radii.
+8. assignGear AI faction to use                                                        | OPTIONAL - string - faction name listed in assignGear.sqf. If this is defined, assignGear AI will be run on the spawned units automatically using this faction.
```

Comment:
the empty string thing should be explicitly mentioned in the comment here

Reply:
None of the existing comments (not written by me) mention this, but done anyway.